### DISCUSSION TITLE
Array column can't support aggregation REPLACE

### DISCUSSION THREAD

**Discussion by gongxingyao** [apache/doris#19180] [A - General / Q&A]
我使用的的doris1.2，能否帮我看下下面的问题
Execution failed: Error Failed to execute sql: java.sql.SQLException: (conn=396) errCode = 2, detailMessage = Array column can't support aggregation REPLACE
CREATE TABLE IF NOT EXISTS kol_brand.dm_brand_youtube_content_info
(
    `video_id` varchar(11) NOT NULL COMMENT "视频id",
    `pub_date` DATETIME DEFAULT "1970-01-01 00:00:00"  COMMENT "视频发布时间",
    `channel_id` VARCHAR(24) COMMENT "视频所属网红id",
    `video_view` BIGINT  COMMENT "视频观看量",
    `likes` INT  COMMENT "视频点赞量",
    `comments` INT COMMENT "视频评论量",
    `brand_ids` ARRAY<INT(11)>  COMMENT "视频的品牌信息旧id",
    `brand_pks` ARRAY<INT(11)>  COMMENT "视频的品牌信息新id"
)
ENGINE=OLAP
UNIQUE KEY(`video_id`,`pub_date`,`channel_id`)
PARTITION BY RANGE(`pub_date`)
(
    PARTITION `p1` VALUES LESS THAN ("2020-01-01 00:00:00"),
    PARTITION `p2` VALUES LESS THAN ("2021-12-31 23:59:59"),
    PARTITION `p3` VALUES LESS THAN ("2022-12-31 23:59:59"),
    PARTITION `p4` VALUES LESS THAN ("2023-12-31 23:59:59"),
    PARTITION `p5` VALUES LESS THAN ("2024-12-31 23:59:59"),
    PARTITION `p6` VALUES LESS THAN ("2025-12-31 23:59:59"),
    PARTITION `p7` VALUES LESS THAN ("2026-12-31 23:59:59"),
    PARTITION `p8` VALUES LESS THAN ("2027-12-31 23:59:59"),
    PARTITION `p9` VALUES LESS THAN ("2028-12-31 23:59:59"),
    PARTITION `p10` VALUES LESS THAN ("2029-12-31 23:59:59"),
    PARTITION `p11` VALUES LESS THAN ("2030-12-31 23:59:59")
)
DISTRIBUTED BY HASH(`video_id`) BUCKETS 1
PROPERTIES (
"replication_num" = "1"
);

**Comment by LemonLiTree** [comment]
https://github.com/apache/doris/pull/17024 
1.2 should not be supported yet
